File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 3/IDiguiseAsInfant.cs
using Phantom_of_the_West.Visual_Novel.Serialization.Event_Flagging;

namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_3
{
	internal class IDiguiseAsInfant : StoryChoice
	{
		internal IDiguiseAsInfant() : base(StoryChoice.GetLocalizedString("ch3IDiguiseAsInfant"), 3025)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch3DroveWithCaitlyn = DriveWithCaitlyn.DisguisedAsInfant;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_3/IDiguiseAsInfant.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_3;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;
import com.christopherraleigh.phantomofthewest.visual_novel.serialization.event_flagging.DriveWithCaitlyn;

/**
 * Created by christopherraleigh on 2016-05-12.
 */
public class IDiguiseAsInfant extends StoryChoice {
    public IDiguiseAsInfant() {
        super(R.string.ch3IDiguiseAsInfant, 3025);
    }

    @Override
    public void select() {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh3DroveWithCaitlyn(DriveWithCaitlyn.DisguisedAsInfant);
        super.select();
    }
}
